R/fetch_eco.R

```diff
@@ -0,0 +1,166 @@
+#' Fetch evidence & conclusion ontology
+#'
+#' Fetches all evidence & conclusion ontology (ECO) information from the EBI database.
```

Comment:
I would maybe describe ECO in a bit more detail so that people know what you can use it for (without having to be familiar with it already). Maybe also link the the website and/or publication.

Reply:
Added more documentation.